Classes/PullRequestReviews/String+Substring.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  String+Substring.swift
+//  Freetime
+//
+//  Created by Joan Disho on 07.05.18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    func substringUntilNewLine() -> String {
+        let substring = self.components(separatedBy: .newlines)[0]
```

Comment:
Can we update this to use `.first` instead so it doesn't crash if there are no newlines? Also a unit test would be 👌